Danger-Swift/Sources/WeTransferPRLinter/WeTransferPRLinter.swift

```diff
@@ -14,4 +16,100 @@ public enum WeTransferPRLinter {
             return
         }
     }
+
+    /// Warn for PRs that are still work in progress.
+    static func validateWorkInProgress(using danger: DangerDSL) {
+        let hasWIPLabel = danger.github.issue.labels.contains(where: { $0.name.contains("WIP") })
+        let hasWIPTitle = danger.github.pullRequest.title.contains("WIP")
+
+        guard !hasWIPLabel, !hasWIPTitle else {
```

Comment:
Because `guard` is used for an early return, I think the code becomes easier to understand if we reverse the check:
```
guard hasWIPLabel, hasWIPTitle  else {
     return  
}

danger.warn("PR is classed as Work in Progress")
```
let me know what you think!

Reply:
Makes sense! I'll use the following code though as we would otherwise check for both WIP Label _and_ title:

```swift
guard hasWIPLabel || hasWIPTitle  else {
     return  
}

danger.warn("PR is classed as Work in Progress")
```

Reply:
That makes sense!